.travis.yml

```diff
@@ -9,17 +9,21 @@ php:
     - 5.6
     - 7.0
     - 7.1
+    - 7.2
     - nightly
 
 matrix:
     fast_finish: true
     allow_failures:
+        - php: 7.2
         - php: nightly
     include:
         - php: 5.5
           env: COMPOSER_FLAGS="--prefer-stable --prefer-lowest"
-        - php: 7.0
+        - php: 7.1
           env: CS_FIXER=run
+        - php: 7.1
+          env: COVERAGE=run
```

Comment:
Can we run only one job for 7.1 with env `CS_FIXER=run` and `COVERAGE=run` ?

Reply:
Yes we can ™